tests/plan.rs

```diff
@@ -5,23 +5,23 @@ const STEAM_DECK: &str = include_str!("./fixtures/linux/steam-deck.json");
 const DARWIN_MULTI: &str = include_str!("./fixtures/darwin/darwin-multi.json");
 
 // Ensure existing plans still parse
-// If this breaks and you need to update the fixture, it's time to raise version.
+// If this breaks and you need to update the fixture, disable these tests, cook bump to a new version, and update the plans.
```

Comment:
"cook bump to a new version" doesn't make sense to me. Is there a better wording that could be used here?

Reply:
Clearly I was hungry~